index.html

```diff
@@ -2,39 +2,145 @@
 <html lang="en">
   <head>
     <title>IdeaBox</title>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,600,500,700' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="styles.css">
   </head>
   <body>
-    <section>
+    <nav>
       <h1>IdeaBox</h1>
+      <hr style="width:100%;text-align:left;margin-left:0;height:2px;background-color:#5356A4;">
       <h2>Filter Starred Ideas</h2>
-      <button>Show Starred Ideas</button>
-    </section>
-    <section>
-      <label>Title</label>
-      <input></input>
-      <label>Body</label>
-      <input></input>
-      <button>Save</button>
-      <button></button>
-      <label>Search ideas</label>
-      <input></input>
-    </section>
-    <section>
-      <div>
-        <div>
-          <button></button>
-          <button></button>
-        </div>
-        <div>
-          <h1>Idea title</h1>
-          <p>Idea body. Don't ever play yourself. Every chance I get, I water the plants, Lion! Cloth talk.</p>
-        </div>
-        <div>
-          <button>Comment</button>
-        </div>
-      </div>
-    </section>
+      <button class='show-starred-ideas-button'>Show Starred Ideas</button>
+      <hr style="width:100%;text-align:left;margin-left:0;height:2px;background-color:#5356A4;">
+      <div class='nav-divider'></div>
+    </nav>
+    <main>
+      <section class="input-form">
+        <label>Title</label>
+        <input></input>
```

Comment:
We need to add the attributes for the labels and inputs like the following: 

```
<label for="fname">First name:</label>
<input type="text" id="fname" name="fname">
```

Reply:
This has been updated!! 🤩